### ISSUE TITLE
DE163 As a data user I want the MNR to collect and provide access to additional information on acoustic deterrent/harassment device at aquaculture sites including information on the ADD types, numbers deployed, rough source levels and deployment schedules (over 24-hr periods continuous/interim) at aquaculture sites so that we can better understand acoustic impacts around aquaculture sites

### ISSUE THREAD

**Issue by LynnHeeley** [jncc/marine-noise-registry#199]
Bit of a pipe dream at this point as would require MSS support or aquaculture industry support (probably on a voluntary basis) but the lack of licensing of ADDs means this information is still missing and really needs to be collected somewhere and I believe the MNR could be a good portal for this and might be less opposed than the EPS licensing route. But also useful to think of what other information can be collected via the MNR for impulsive activities that might inform our understanding of underwater noise pollution other than the SEL/SPL.

**Comment by robanga-jncc** [comment]
Agreed on the importance of this data but there are challenges on how this could be acquired. 
It could be an additional data set that is held within the bigger project which would help with sharing the data. The question as to whether it needs to be public or private would depend on who is uploading the data. If regulators agreed to share datasets from their own projects, but could be challenging if needs to be private.

**Comment by TetrienneBox** [comment]
Group 3 - disagree with movement to nice to have. 
should be Phase 2. - we need the data to help assessments as a data user.
